<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>501</width>
    <height>832</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QVTKWidget" name="qvtkWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>500</height>
         </size>
        </property>
        <property name="title">
         <string>Controls</string>
        </property>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>141</width>
           <height>111</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QGroupBox { 
     border: 1px solid black; 
     border-radius: 1px; 
 } </string>
         </property>
         <property name="title">
          <string>STL view</string>
         </property>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>41</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Opacity</string>
          </property>
         </widget>
         <widget class="QSlider" name="horizontalSlider">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>40</y>
            <width>131</width>
            <height>29</height>
           </rect>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="value">
           <number>50</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
         <widget class="QDoubleSpinBox" name="doubleSpinBox">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>20</y>
            <width>51</width>
            <height>21</height>
           </rect>
          </property>
          <property name="maximum">
           <double>1.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
          <property name="value">
           <double>0.500000000000000</double>
          </property>
         </widget>
         <widget class="QCheckBox" name="checkBox">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>60</y>
            <width>83</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>Wireframe</string>
          </property>
         </widget>
         <widget class="QCheckBox" name="checkBox_2">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>80</y>
            <width>101</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>Edges visible</string>
          </property>
         </widget>
        </widget>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>20</y>
           <width>120</width>
           <height>80</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QGroupBox { 
     border: 1px solid black; 
     border-radius: 1px; 
 } </string>
         </property>
         <property name="title">
          <string>Camera</string>
         </property>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>31</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>FSAA</string>
          </property>
         </widget>
         <widget class="QSpinBox" name="spinBox">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>20</y>
            <width>53</width>
            <height>21</height>
           </rect>
          </property>
          <property name="maximum">
           <number>12</number>
          </property>
         </widget>
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>50</y>
            <width>61</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Cam angle</string>
          </property>
         </widget>
         <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>50</y>
            <width>51</width>
            <height>20</height>
           </rect>
          </property>
         </widget>
        </widget>
        <widget class="QGroupBox" name="groupBox_5">
         <property name="geometry">
          <rect>
           <x>300</x>
           <y>20</y>
           <width>161</width>
           <height>181</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QGroupBox { 
     border: 1px solid black; 
     border-radius: 1px; 
 } </string>
         </property>
         <property name="title">
          <string>HW Control</string>
         </property>
         <widget class="QLabel" name="label_10">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>50</y>
            <width>53</width>
            <height>14</height>
           </rect>
          </property>
          <property name="text">
           <string>Reference</string>
          </property>
         </widget>
         <widget class="QComboBox" name="comboBox_2">
          <property name="geometry">
           <rect>
            <x>70</x>
            <y>50</y>
            <width>71</width>
            <height>21</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_11">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>80</y>
            <width>53</width>
            <height>14</height>
           </rect>
          </property>
          <property name="text">
           <string>Tool</string>
          </property>
         </widget>
         <widget class="QComboBox" name="comboBox_3">
          <property name="geometry">
           <rect>
            <x>70</x>
            <y>80</y>
            <width>71</width>
            <height>21</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_13">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>100</y>
            <width>53</width>
            <height>14</height>
           </rect>
          </property>
          <property name="text">
           <string>Catheter</string>
          </property>
         </widget>
         <widget class="QComboBox" name="comboBox_5">
          <property name="geometry">
           <rect>
            <x>70</x>
            <y>100</y>
            <width>71</width>
            <height>21</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_14">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>120</y>
            <width>71</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>updates/sec.</string>
          </property>
         </widget>
         <widget class="QSpinBox" name="spinBox_2">
          <property name="geometry">
           <rect>
            <x>90</x>
            <y>120</y>
            <width>53</width>
            <height>24</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_4">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>53</width>
            <height>14</height>
           </rect>
          </property>
          <property name="text">
           <string>Port</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="lineEdit">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>20</y>
            <width>91</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>/dev/ttyUSB0</string>
          </property>
         </widget>
         <widget class="QPushButton" name="pushButton">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>150</y>
            <width>85</width>
            <height>27</height>
           </rect>
          </property>
          <property name="text">
           <string>Run</string>
          </property>
         </widget>
        </widget>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="geometry">
          <rect>
           <x>170</x>
           <y>110</y>
           <width>120</width>
           <height>80</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QGroupBox { 
     border: 1px solid black; 
     border-radius: 1px; 
 } </string>
         </property>
         <property name="title">
          <string>Tool view</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <widget class="QLabel" name="label_5">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>31</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>FSAA</string>
          </property>
         </widget>
         <widget class="QSpinBox" name="spinBox_3">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>20</y>
            <width>53</width>
            <height>21</height>
           </rect>
          </property>
          <property name="maximum">
           <number>12</number>
          </property>
         </widget>
         <widget class="QDoubleSpinBox" name="doubleSpinBox_3">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>50</y>
            <width>51</width>
            <height>20</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_6">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>50</y>
            <width>61</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Cam angle</string>
          </property>
         </widget>
        </widget>
        <widget class="QGroupBox" name="groupBox_6">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>210</y>
           <width>451</width>
           <height>211</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QGroupBox { 
     border: 1px solid black; 
     border-radius: 1px; 
 } </string>
         </property>
         <property name="title">
          <string>Sensors</string>
         </property>
         <widget class="QGroupBox" name="groupBox_7">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>30</y>
            <width>161</width>
            <height>161</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QGroupBox::title { 
    background-color: transparent;
     subcontrol-position: top left; /* position at the top left*/ 
     padding:2 13px;
 } </string>
          </property>
          <property name="title">
           <string>GroupBox</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="1" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Tx</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QLineEdit" name="lineEdit_6"/>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="lineEdit_3"/>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="lineEdit_4"/>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>Q0</string>
             </property>
            </widget>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="label_15">
             <property name="text">
              <string>Qy</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Tx</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label_17">
             <property name="text">
              <string>Qx</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="lineEdit_2"/>
           </item>
           <item row="6" column="1">
            <widget class="QLineEdit" name="lineEdit_7"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Ty</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLineEdit" name="lineEdit_5"/>
           </item>
           <item row="7" column="0">
            <widget class="QLabel" name="label_16">
             <property name="text">
              <string>Qz</string>
             </property>
            </widget>
           </item>
           <item row="7" column="1">
            <widget class="QLineEdit" name="lineEdit_8"/>
           </item>
           <item row="0" column="1">
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>600</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>501</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionLoad_STL_mesh"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLoad_STL_mesh">
   <property name="text">
    <string>Load STL mesh</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
